LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2022
+COPYRIGHT HOLDER: Shelly
```

Comment:
Do you want me to remove the CCO license and switch the text to the MIT license? Is this copyright required and do you want to include your last name here? Here is an open issue for this topic that we can close after we finish making edits to the license information: https://github.com/USEPA/TADA/issues/63